boards/firefly/Makefile.include

```diff
@@ -0,0 +1,7 @@
+USEMODULE += remote-common
+
+# define the default port depending on the host OS
+PORT_LINUX  ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
```

Comment:
Are you sure this is the correct port for mac OS ? can someone with a mac can confirm ?
on remote-pa (cc2538), it's `/dev/tty.usbserial`, on remote-reva/b it's also `/dev/tty.SLAB_USBtoUART`

Reply:
I am on Linux so would also be happy for someone on MacOS to check.

Reply:
typically that name depends on the USB2Serial converted used. According to the [data sheets](https://github.com/Zolertia/Resources/wiki/Zolertia-Technical-documentation#the-re-mote-platform-documentation) both (remote and firefly) use the CP2104, so I would assume names are also the same. Though I can't verify as I have no Firefly (only remotes) available 😞 

TL;DR leave as is, we'll fix if needed.